api/src/main/java/org/ccci/idm/user/Dn.java

```diff
@@ -16,13 +16,11 @@
 public class Dn implements Serializable {
     private static final long serialVersionUID = 5510344429904560934L;
 
+    public static final Dn ROOT = new Dn();
+
     @Nonnull
     private final List<Component> components;
 
-    public Dn() {
-        components = ImmutableList.of();
-    }
-
```

Comment:
also, both constructors are used in various places within this project based on needs